src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -45,9 +48,15 @@
             @ApiResponse(code = 404, message = "An actor with the specified ID was not found") })
     public Mono<ResponseEntity<Actor>> getActor(
             @ApiParam(value = "The ID of the actor to look for", example = "nm0000002", required = true) @PathVariable("id") String actorId) {
-        return actorsDao.getActorById(actorId)
-                .map(savedActor -> ResponseEntity.ok(savedActor))
-                .defaultIfEmpty(ResponseEntity.notFound().build());
+
+        if(validator.validActorId(actorId)) {
```

Comment:
Consider naming this function as isValidActorId - makes it a bit more readable

Reply:
Done